avocado/utils/linux_modules.py

```diff
@@ -15,11 +15,17 @@
 # Original author: Ross Brattain <ross.b.brattain@intel.com>
 
 """
-APIs to list and load/unload linux kernel modules.
```

Comment:
I don't think it's necessary to list all the APIs. One can look for per-section descriptions so somethink like `APIs related to linux kernel modules` would be IMO ideal...

Anyway both are fine, just one issue with this one you should not report `0, 1, 2`, but you should always use the names. People are not suppose to use the values as they can change.